fs/fs.go

```diff
@@ -0,0 +1,644 @@
+// Provides Filesystem Stats
+package fs
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+	"sync"
+	"syscall"
+	"time"
+
+	"github.com/docker/docker/pkg/mount"
+	"github.com/fsouza/go-dockerclient"
+	"github.com/intelsdi-x/snap-plugin-collector-docker/config"
+	"github.com/intelsdi-x/snap-plugin-collector-docker/mounts"
+	"github.com/intelsdi-x/snap-plugin-collector-docker/wrapper"
+	zfs "github.com/mistifyio/go-zfs"
+)
+
+const (
+	LabelSystemRoot = "root"
```

Comment:
exported const LabelSystemRoot should have comment (or a comment on this block) or be unexported